crates/typst-pdf/src/catalog.rs

```diff
@@ -87,8 +87,20 @@ pub fn write_catalog(
         xmp.pdf_keywords(&joined);
     }
 
-    let date = ctx.document.info.date.unwrap_or(ctx.options.timestamp);
-    let tz = ctx.document.info.date.is_auto();
+    // First, try to use the date from the document information, and it can
+    // be `none`. Second, try to use the date from the options. If neither is
```

Comment:
```suggestion
    // be `auto`. Second, try to use the date from the options. If neither is
```

Reply:
What I actually want to express here is that if the `document.date` is set to specific `datetime` or `none`, it will be adapted first. And if it is `auto`, then the alternative cases will be considered. Maybe I should improve this comment.